drivers/pinmux/stm32/pinmux_board_stm32f429i_disc1.c

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
@galak However, I see all ST Micro's board pinmux files are here. Is there a decision to move each of them back to each board directory?